src/main/java/com/pizzashop/pizzashop/Controllers/PizzasController.java

```diff
@@ -0,0 +1,67 @@
+package com.pizzashop.pizzashop.Controllers;
+
+import com.pizzashop.pizzashop.Models.Pizza;
+import com.pizzashop.pizzashop.Repositories.IngredientsRepository;
+import com.pizzashop.pizzashop.Repositories.PizzasRepository;
+import com.pizzashop.pizzashop.Repositories.PizzasTypesRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+@RestController
+@RequestMapping("/pizzas")
+public class PizzasController {
+    private PizzasRepository pizzasRepository;
+
+    @Autowired
+    private PizzasTypesRepository pizzasTypesRepository;
+
+    @Autowired
+    private IngredientsRepository ingredientsRepository;
+
+    public PizzasController(PizzasRepository pizzasRepository){
+        this.pizzasRepository = pizzasRepository;
+    }
+
+    @GetMapping
+    public Iterable<Pizza> allPizzas(){
+        Iterable<Pizza> pizzas =this.pizzasRepository.findAll();
```

Comment:
@DanielIllanes, One more thing... I don't really understand what are you doing here. Let's have a quick call to discuss abou it.